cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala

```diff
@@ -69,6 +69,14 @@ class AggregatingFunctionsTest extends DocumentingTest {
         #The latter, `+count(*)+` is an aggregate expression.
         #The matching paths will be divided into different buckets, depending on the grouping key.
         #The aggregate function will then be run on these buckets, calculating an aggregate value per bucket.""".stripMargin('#'))
+    p("""The input expression of an aggregation function can contain non-grouping variables, for example `+sum(1 + n.x)+`.
```

Comment:
Perhaps it should read:
> The input expression of an aggregation function can contain any expression, including expressions that are not grouping keys. 
> However, not all expressions can be _composed_ with aggregation functions.
> The example below will throw an error since we compose `n.x`, which is not a grouping key, with the aggregation expression `+count(*)+`.
> For more information see <<grouping-keys,Grouping keys>>.

I.e. we don't have to be very specific here (the details are explained in the link) but rather just introduce the idea that some expressions don't go together with aggregation functions.